### ISSUE TITLE
Shell Integration for Windows Terminal

### ISSUE THREAD

**Issue by Herbie34** [facundoolano/rpg-cli#149]
I have ported `example.sh` for Windows Terminal. There are a few differences from the original version:
- I couldn’t figure out how to overwrite `cd`, so I used `cdir` instead.
- To avoid hitting the absolute path length limit, I made the created directories single-digit numbers.
- It seems that `&&` can be used in PowerShell 7, but it’s not available in version 5, so I omitted the error handling.

Microsoft.PowerShell_profile.ps1 ($PROFILE)
```PowerShell
Set-Variable -Option Constant -Name RPG -Value "C:\your\path\to\rpg-cli.exe"

function rpg() {
    & $RPG $args
    sync_rpg
}

function cdir() {
    & $RPG cd $args
    sync_rpg
}

function dn() {
    $current = (Get-Item $PWD).BaseName
    if ($current -match "^[0-9]+$") {
        $next = (([int]$current) + 1) % 10
        New-Item -ItemType Directory -ErrorAction SilentlyContinue $next > $null
        cdir $next
    } elseif (Test-Path "1") {
        cdir 1
    } else {
        New-Item -ItemType Directory -ErrorAction SilentlyContinue "dungeon\1" > $null
        cdir "dungeon/1"
    }
    rpg ls
}

function sync_rpg() {
    $pwd = & $RPG pwd
    Set-Location -Path $pwd
}
```

**Comment by facundoolano** [comment]
Can you send a pull request to add this to the shell dir?

**Comment by facundoolano** [comment]
Closed by #150
